### PR TITLE
Upgrade Platform.sh configuration snippet.

### PR DESCRIPTION
**Pull request by GuGuss**
Following a recent Platform.sh upgrade: https://docs.platform.sh/user_guide/reference/upgrade/index.html

### CODE REVIEW

File: cookbook/deployment/platformsh.rst [hunk 1]
```diff
@@ -37,9 +37,11 @@ Platform.sh how to deploy your application (read more about
 
     # The name of this app. Must be unique within a project.
     name: myphpproject
-
-    # The toolstack used to build the application.
-    toolstack: "php:symfony"
+    
+    # The type of the application to build.
+    type: php:5.4
```
Comment (xabbuh) [inline cookbook/deployment/platformsh.rst]: Giving a version number here does work? Just asking as it is not explained in the docs.

Comment (WouterJ) [inline cookbook/deployment/platformsh.rst]: if it does, I would vote to use a newer version if possible (although no PHP is used by this project, it's quite nice to show a good practice not using versions that reached eol)

**Comment by GuGuss** [comment]
This is documented here: https://docs.platform.sh/user_guide/reference/platform-app-yaml.html#type-and-build.

Platform.sh can currently build 3 of the following PHP versions: 5.4, 5.5, 5.6.

Thanks for the review guys!

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you Augustin.

**Pull request closed without merging**